modules/menuing.py

```diff
@@ -299,7 +297,8 @@ def select_mon(self):
         if self.game in ["POKEMON EMER", "POKEMON FIRE", "POKEMON LEAF"]:
             while task_is_active("TASK_HANDLECHOOSEMONINPUT"):
                 context.emulator.press_button("A")
-                yield
+                # yield
+                context.emulator.run_single_frame()
```

Comment:
Any reason you're running the frame instead of yielding?

Reply:
It randomly breaks when yielding. Works about 50% with yield, 100% with single frame